### PR TITLE
Added custom contact

### PR DESCRIPTION
**Pull request by JaySunSyn**
Thanks! :)

```
<paper-contact-custom icon="icons:favorite" no-link>Love it!</paper-contact-custom>
```

### CODE REVIEW

File: paper-contact-custom.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../iron-icon/iron-icon.html">
+<link rel="import" href="../iron-icons/communication-icons.html">
+<link rel="import" href="../paper-item/paper-icon-item.html">
+<link rel="import" href="../paper-ripple/paper-ripple.html">
+<link rel="import" href="paper-contact-behavior.html">
+<link rel="import" href="paper-contact-shared-styles.html">
+
+<!--
+Component for email contact details, that triggers the opening of the email client on tap.
```
Comment (ronnyroeller) [inline paper-contact-custom.html]: Could you please adjust docu

File: paper-contact-custom.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+<script>
+(function() {
+	Polymer({
+		is: 'paper-contact-custom',
+
+		behaviors: [
+			Polymer.PaperContactBehavior
+		],
+
+        properties: {
```
Comment (ronnyroeller) [inline paper-contact-custom.html]: Please double check indentation. It should be tabs (instead of spaces).

File: paper-contact-custom.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                type: String,
+            },
+
+            target: {
+                type: String,
+            }
+        },
+
+		// Private methods
+        _open: function() {
```
Comment (ronnyroeller) [inline paper-contact-custom.html]: Please check indentation. It should be tabs (instead of spaces).

**Review by ronnyroeller — CHANGES_REQUESTED** [review]

**Comment by JaySunSyn** [comment]
I've added linter files. You can now run `npm run lint` which makes contribution easier..
Please adjust .eslintrc.json to your coding style if you decide to keep it.

**Comment by JaySunSyn** [comment]
You could add a `.travis` file after `.eslintrc.json` adjusted. (to let the build process fail if test or linter fails)

Example for `.travis`:

```
language: node_js
dist: trusty
sudo: required
addons:
  firefox: "latest"
  apt:
    sources:
      - google-chrome
    packages:
      - google-chrome-stable
node_js:
  - "node"
  - "6"
cache:
  directories:
    - node_modules
install:
  - npm install -g polymer-cli
  - npm install
  - polymer install
before_script:
  - npm run lint
script:
  - xvfb-run npm run test
  - xvfb-run npm run test:integration
```

**Comment by ronnyroeller** [comment]
Thanks for the adjustments, @JaySunSyn 
Would you mind splitting the ESlint part into a separated PR (so we have the PR6 only cover the new feature without changing the build process)? Thanks.

**Pull request closed without merging**